README.md

```diff
@@ -386,6 +386,10 @@ _[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software_
   * [twister](http://twister.net.co/) - Fully decentralized P2P microblogging platform leveraging  the free software implementations of Bitcoin and BitTorrent protocols. ([Source Code](https://github.com/miguelfreitas/twister-core)) `BSD/MIT` `C++`
   * [Vanilla Forums](https://vanillaforums.org/) - Simple and flexible forum software. ([Source Code](https://github.com/vanilla/vanilla)) `GPLv2` `PHP`
 
+### Instant Messaging
+
+  * [Spectrum 2](http://spectrum.im/) - Spectrum 2 is an open source instant messaging transport.  It allows users to chat together even when they are using different IM networks.
```

Comment:
* Please add license and language tags
 * Please add a direct link to the source code
 * Please move it to https://github.com/Kickball/awesome-selfhosted#custom